src/netcdf_io/calc_analysis.fd/inc2anl.f90

```diff
@@ -277,12 +277,14 @@ subroutine add_increment(fcstvar, incvar)
           incncfile = open_dataset(incr_file, paropen=.true.)
           ! JEDI-derived increments have a time dimension, so read to appropriate array
           if ( jedi ) then
-             call read_vardata(incncfile, trim(incvar)//"_inc", work3d_inc_jedi, nslice=k, slicedim=3)
-             ! add increment to background
-             do j=1,nlat
-                jj=nlat+1-j ! increment is S->N, history files are N->S
-                work3d_bg(:,j,1) = work3d_bg(:,j,1) + work3d_inc_jedi(:,jj,1)
-             end do
+             if (has_var(incncfile, trim(incvar)//"_inc")) then
```

Comment:
Should this check if specific variables are missing, similar to how `interp_inc` now allows for only the hydrometeors to be missing?

Reply:
One could limit this check to the specific hydrometeors flagged in the revised `driver.F90`.  A more general approach was taken here to exclude all variables for which no increment is found.  What approach shall we take?  Thoughts @CatherineThomas-NOAA , @emilyhcliu , @CoryMartin-NOAA

Reply:
The best way is probably to relate the required increments from the control variable table in anavinfo.  This requires a bit more coding and related script changes.  For now, the proposed change should work and be acceptable.

Reply:
Thank you @emilyhcliu for your suggestion.  I agree with you.  We can stick with the easier solution for the time being.

Reply:
Logical variable `readinc` added to improve runtime printout.  When a JEDI increment file is processed and a specified increment field is not present `readinc` is set to `.false.`.  The calling routine uses the `jedi` logical along with `readinc` to select the appropriate runtime message.  If the increment is added to the background the runtime printout is, for example,
```
nid002740.dogwood.wcoss2.ncep.noaa.gov 0:  Adding Increment to spfh   sphum
nid002740.dogwood.wcoss2.ncep.noaa.gov 0:  Adding Increment to tmp    T
nid002740.dogwood.wcoss2.ncep.noaa.gov 0:  Adding Increment to ugrd   u
```
Otherwise the printout is
```
nid002740.dogwood.wcoss2.ncep.noaa.gov 0:  Copying from Background grle
nid002740.dogwood.wcoss2.ncep.noaa.gov 0:  Copying from Background rwmr
nid002740.dogwood.wcoss2.ncep.noaa.gov 0:  Copying from Background snmr
```

Previously, even though JEDI increments for grle, rwmr, and snmr were not available, the runtime printout was
```
nid001756.dogwood.wcoss2.ncep.noaa.gov 0:  Adding Increment to grle   grle
nid001756.dogwood.wcoss2.ncep.noaa.gov 0:  Adding Increment to rwmr   rwmr
nid001756.dogwood.wcoss2.ncep.noaa.gov 0:  Adding Increment to snmr   snmr
```
This change committed at 1c512a1.